ground-control/.env.example

```
@@ -25,3 +25,12 @@ TLS_CERT_FILE=
 TLS_KEY_FILE=
 # Set TLS_CA_FILE to enable mutual TLS (client certificate verification)
 TLS_CA_FILE=
+
+# Audit logging for security events (optional, disabled by default)
+# Set AUDIT_LOG_ENABLED=true to write authentication, registration, user
+# management, and policy events to a structured JSON log with rotation.
+AUDIT_LOG_ENABLED=false
+AUDIT_LOG_PATH=./audit.log
+AUDIT_LOG_MAX_SIZE_MB=100
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUDIT_LOG_MAX_SIZE_MB key should go before the AUDIT_LOG_PATH key

(UnorderedKey)